sycl/test/lit.site.cfg.py.in

```diff
@@ -2,18 +2,16 @@
 
 import sys
 
-config.clang = "@CLANG_IN_BUILD@"
-config.clangxx = "@CLANGXX_IN_BUILD@"
-config.clang_cl = "@CLANGCL_IN_BUILD@"
 config.llvm_tools_dir = "@LLVM_TOOLS_DIR@"
+config.sycl_tools_dir = lit_config.params.get('SYCL_TOOLS_DIR', "@LLVM_TOOLS_DIR@")
 config.lit_tools_dir = "@LLVM_LIT_TOOLS_DIR@"
-config.llvm_build_libs_dir = "@LLVM_BUILD_LIBRARY_DIRS@"
-config.llvm_build_bins_dir = "@LLVM_BUILD_BINARY_DIRS@"
-config.llvm_binary_dir = "@LLVM_BINARY_DIR@"
-config.sycl_include = "@SYCL_INCLUDE@"
+config.sycl_include = lit_config.params.get('SYCL_INCLUDE', "@SYCL_INCLUDE@")
 config.sycl_obj_root = "@SYCL_BINARY_DIR@"
 config.sycl_source_dir = "@SYCL_SOURCE_DIR@/source"
 config.opencl_libs_dir = os.path.dirname("@OpenCL_LIBRARIES@")
+config.sycl_libs_dir = lit_config.params.get('SYCL_LIBS_DIR', "@LLVM_LIBS_DIR@")
+config.target_triple = "@TARGET_TRIPLE@"
+config.host_triple = "@LLVM_HOST_TRIPLE@"
```

Comment:
I cannot find direct uses of these new options, are they needed?

Reply:
`llvm_config.use_clang()` function call requires them to be set.

Reply:
@AlexeySachkov, please, "resolve" conversation.